rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -165,21 +164,6 @@ public Response actionWithLRA(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI contextI
             return getResult(cancel, contextId);
         }
 
-        @GET
-        @Path("time-limit")
```

Comment:
Would you consider to have the test `time-limit` left in the code even there will be no reaper used?

Reply:
That is part of `git revert` for the reaper change.
We have timeout tests in the TCK as well.

Reply:
But you are right, if I deemed it useful then, then it is still useful. Thanks for picking up on it.

Note that the reason I have included a number of unit tests with the coordinator is becuase they generally complement the TCK and they also directly test coordinator functionality so they are handy to have when making coordinator specific changes.

Reply:
+1